crml/staking/src/lib.rs

```diff
@@ -686,7 +678,8 @@ where
 	T::ValidatorIdOf: Convert<<T as frame_system::Config>::AccountId, Option<<T as frame_system::Config>::AccountId>>,
 {
 	fn disable_validator(validator: &<T as frame_system::Config>::AccountId) -> Result<bool, ()> {
-		<pallet_session::Module<T>>::disable(validator)
+		// CHECK
+		Ok(<pallet_session::Module<T>>::disable(validator))
```

Comment:
Could we change the return type from Result here?

Reply:
the trait comes from substrate so we just follow what's required there